spec/features/application_letters_spec.rb

```diff
@@ -149,6 +149,26 @@
     expect(page).to have_text('Bewerbung erstellen')
   end
 
+  %i[coach organizer].each do |role|
+    it "logged in as #{role} I cannot see personal details" do
+      login(role)
+      expect(page).to_not have_text(@application_letter.user.profile.address)
+      expect(page).to_not have_text(@application_letter.user.profile.school)
+    end
+  end
+
+  it "logged in as admin I can see personal details" do
+    login(:admin)
+    expect(page).to have_text(@application_letter.user.profile.address)
```

Comment:
No, the school is not actually displayed. never. Tobi just wanted to make sure, it also never will be in the future. (See US comments)